src-docs/src/views/guidelines/toasts.js

```diff
@@ -13,478 +13,601 @@ import {
+              style={{ maxWidth: 300 }}
+              title="Your object has been saved"
+            />
+          </GuideRuleExample>
+        </GuideRule>
+
+        <GuideRule
+          description="Don't include the word &quot;successfully.&quot; It's implied."
+        >
+          <GuideRuleExample
+            panel={false}
+            frame
+            type="do"
+            text="Do. Use this format for a success message."
+          >
+            <EuiToast
+              color="success"
+              style={{ maxWidth: 300 }}
+              title="Dashboard 'My_dashboard' was saved"
+            />
+
+          </GuideRuleExample>
+
+          <GuideRuleExample
+            type="dont"
+            panel={false}
+            frame
+            text="Don&apos;t include &quot;successfully.&quot;"
+          >
+            <EuiToast
+              color="success"
+              style={{ maxWidth: 300 }}
+              title="Dashboard 'My_dashboard' was successfully saved"
+            />
+          </GuideRuleExample>
+        </GuideRule>
+
+        <GuideRule
+          description="For a message about multiple objects, include the object count, but not the names of the objects."
+        >
+          <GuideRuleExample
+            panel={false}
+            type="do"
+            frame
+            text="Do. Include the object count."
+          >
+            <EuiToast
+              color="success"
+              style={{ maxWidth: 300 }}
+              title="4 visualizations were deleted"
+            />
+
+          </GuideRuleExample>
+          <GuideRuleExample
+            panel={false}
+            type="dont"
+            frame
+            text="Don't overwhelm the user by listing the names of all the objects."
+          >
+            <EuiToast
+              color="success"
+              style={{ maxWidth: 300 }}
+              title="Visualization 1, Visualation 2, Visualization 3, and Visualization 4 were deleted"
+            />
+
+          </GuideRuleExample>
+        </GuideRule>
+
+        <GuideRuleTitle>Use call-to-action buttons when the content needs more room</GuideRuleTitle>
+
+        <GuideRule
+          description="Occassionally the content of a toast is too involved to fit into the constrained space of a toast.
+          This is common in long error messages. In these cases use the test message to deliver the summary of the
+          information and use a button to provide a call-to-action for the full message."
+        >
+          <GuideRuleExample
+            panel={false}
+            type="do"
+            frame
+            text="Utlize buttons, modals and secondary pages to deliver long form content when toasts contain detailed content"
+          >
+            <EuiToast
+              style={{ maxWidth: 300 }}
+              color="danger"
+              title="Your visualization has an error"
+            >
+              <p>The maximum bucket size of 200 was exceeded</p>
```

Comment:
Apparently we're using periods now ;)
```suggestion
              <p>The maximum bucket size of 200 was exceeded.</p>
```